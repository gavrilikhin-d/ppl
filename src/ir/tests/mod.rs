use crate::test_compilation_result;

test_compilation_result!(empty_constructor);
