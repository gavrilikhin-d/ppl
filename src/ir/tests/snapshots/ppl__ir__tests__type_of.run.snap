---
source: src/ir/tests/mod.rs
expression: run_log
---
Integer
